docs/compliance/pci/index.md

```diff
@@ -0,0 +1,366 @@
+
+- [End-to-End-SSL] (https://docs.microsoft.com/en-us/azure/application-gateway/application-gateway-end-to-end-ssl-powershell)
+- Enable [SSL Offload](https://docs.microsoft.com/en-us/azure/application-gateway/application-gateway-ssl-portal)
+- Disable [TLS v1.0 and v1.1](https://docs.microsoft.com/en-us/azure/application-gateway/application-gateway-end-to-end-ssl-powershell)
+- [Web application firewall](https://docs.microsoft.com/en-us/azure/application-gateway/application-gateway-webapplicationfirewall-overview) (WAF mode)
+- [Prevention mode](https://docs.microsoft.com/en-us/azure/application-gateway/application-gateway-web-application-firewall-portal) with OWASP 3.0 ruleset
+- Enable [diagnostics logging](https://docs.microsoft.com/en-us/azure/application-gateway/application-gateway-diagnostics)
+- [Custom health probes](https://docs.microsoft.com/en-us/azure/application-gateway/application-gateway-create-gateway-portal)
+- [Azure Security Center](https://azure.microsoft.com/en-us/services/security-center) and [Azure Advisor](https://docs.microsoft.com/en-us/azure/advisor/advisor-security-recommendations) provide additional protection and notifications. Azure Security Center also provides a reputation system.
+
+#### Virtual network
+
+The foundational architecture defines a private virtual network with an address space of 10.0.0.0/16.
+
+#### Network security groups
+
+Each of the network tiers has a dedicated network security group (NSG):
+- A DMZ network security group for firewall and Application Gateway WAF
+- An NSG for management jumpbox/bastion host 
+- An NSG for the app service environment
+
+Each of the NSGs have specific ports and protocols opened for the secure and correct operation of the solution. For more information, see [PCI Guidance - Network Security Groups](#network-security-groups).
+
+Each of the NSGs have specific ports and protocols opened for the secure and
+correct working of the solution. In addition, the following configurations are enabled for each NSG:
+- Enabled [diagnostic logs and events](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-nsg-manage-log) are stored in storage account 
+- Connected OMS Log Analytics to the [NSG's diagnostics](https://github.com/krnese/AzureDeploy/blob/master/AzureMgmt/AzureMonitor/nsgWithDiagnostics.json)
+
+#### Subnets
+ Ensure each subnet is associated with its corresponding NSG.
+
+#### Custom domain SSL certificates
+ HTTPS traffic is enabled using a custom domain SSL certificate.
+
+### Data at rest
+To meet encrypted data-at-rest requirements, all [Azure Storage](https://azure.microsoft.com/en-us/services/storage/) uses the following:
+
+#### Azure Storage
+- [Storage Service Encryption](https://docs.microsoft.com/en-us/azure/storage/storage-service-encryption)
```

Comment:
Avoid single-item lists. Consider rephrasing this as a sentence - the Azure Storage accounts are encrypted using Azure Storage Encryption